ur_description/cfg/view_robot.rviz

```diff
@@ -151,7 +159,7 @@ Visualization Manager:
   Views:
     Current:
       Class: rviz_default_plugins/Orbit
-      Distance: 10
+      Distance: 2.8069188594818115
```

Comment:
This could be also reset since it is possibly an unintentional change.

Reply:
[Fix](https://github.com/PickNikRobotics/Universal_Robots_ROS2_Driver/pull/14/commits/90ae21d7a8b96abf46cd207bd73bbc7582b2eb39)